### PR TITLE
lookup storage domains and pods

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
cc @SimonLorentz @fgoebel @cmeissner @m-bucher

File: plugins/module_utils/foreman_helper.py [hunk 1]
```diff
@@ -820,6 +820,36 @@ def find_network(self, name, compute_resource, cluster=None):
             self.fail_json(msg=err_msg)
         return network
 
+    def find_storage_domain(self, name, compute_resource, cluster=None):
```
Comment (evgeni) [inline plugins/module_utils/foreman_helper.py:823]: at this point, the four find methods look really similar, so probably should be dried up.

**Review by ehelms — APPROVED** [review]
Tests look good, if you want to DRY it up go for it

**Comment by evgeni** [comment]
> Tests look good, if you want to DRY it up go for it

I'll do that in a follow up PR, thanks

**Pull request merged**